packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -147,6 +147,46 @@ export const useChatMessageRenderStyles = makeStyles({
   }
 });
 
+const bodyWithPlaceholderImage = (myMessage: boolean = false): GriffelStyle => {
```

Comment:
Please add styles to `useChatMyMessageStyles` instead as we shouldn't use `GriffelStyle` directly
The change probably can be part of `body` item

![image](https://github.com/Azure/communication-ui-library/assets/98852890/662d470c-0703-484d-97d5-9513a9bac0a4)

Reply:
it looks like the separate style was created to share between my message and message styles. Please add the styles to `useChatMessageCommonStyles` instead